### PR TITLE
`complete task 02 with setup from task 01

### PR DESCRIPTION
**Pull request by JoeAAdams**
completes task 02 with tests

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+describe("/api", () => {
+    describe("GET", () => {
+        describe("/topics", () => {
+            test("all results should contain correct keys", () => {
+                return request(app)
+                .get("/api/topics")
+                .expect(200)
+                .then(({body}) => {
+                    const { topics } = body
+                    topics.forEach(topic => {
```
Comment (nc-roisin-oreilly) [inline __tests__/app.test.js:24]: if you're iterating through an array in your tests, you must include a line in your test to check that the array isn't empty (eg. checking the length isn't 0), otherwise this test will give you a false positive

File: db/app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const { getTopics } = require('./Controllers/topics _controllers');
+const app = express();
+
+
+app.use(express.json());
```
Comment (nc-roisin-oreilly) [inline db/app.js:6]: you're not using this line yet, take it out until you actually need it please

File: db/app.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+app.get('/api/topics', getTopics);
+
+app.use((err,req,res,next)=>{
```
Comment (nc-roisin-oreilly) [inline db/app.js:10]: nicely done

**Review by nc-roisin-oreilly — COMMENTED** [review]
looks great so far! I've left a couple of comments for some changes but otherwise keep it up :)
